### DISCUSSION TITLE
Repository creation

### DISCUSSION THREAD

**Discussion by VedantWatpade** [community/community#136940] [Repositories] [answered]
### Select Topic Area

Show & Tell

### Body

![Screenshot 2024-08-27 174821](https://github.com/user-attachments/assets/426b64cc-4ea6-4105-9457-63e20413293b)
 
when i click the "create new repository"  it show creation fail but when i click the button new it creat the new repository why see the two buttons i have circled there and the red line on top message that creation fail

**Comment by dfosco** [comment, marked as answer]
Hello @VedantWatpade! 

That definitely has to be made clearer on the UI, but it's possible you forgot to add the repository name. It's a mandatory field to create the repository from the button on the left.

Let me know if that works!

**Reply by VedantWatpade** [reply]
ohh !  sorry sorry. i completely forgot about that im sorry

**Reply by SonuG56** [reply]
SonuG56

**Reply by Abusayed325** [reply]
right

**Comment by theshawnv** [comment]
In this scenario, you must add your repository name as it's required.

**Reply by VedantWatpade** [reply]
ohh !  sorry sorry. i completely forgot about that im sorry
